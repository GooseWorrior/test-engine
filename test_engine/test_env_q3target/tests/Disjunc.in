|

a
q
||

a
q
a|b

a
b
ab
q
ab|c
ab
c
a
ac
q
(ab)c|d
abc
d
abd
ab
q
(a|b)c
a
b
ac
bc
abc
c
q
a|b|c
a
b
c
ab
abc
q
(a|b|c)d
ad
bd
cd
a
b
c
d
q
(ab|c)d
abd
cd
abd
acb
abcd
q
a(b|c)
ab
ac
a
b
c
q

