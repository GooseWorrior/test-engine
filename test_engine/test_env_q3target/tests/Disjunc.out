Disjunction (,)
true
false
Disjunction (,Disjunction (,))
true
false
Disjunction (a,b)
false
true
true
false
Disjunction (Concatenation (a,b),c)
true
true
false
false
Disjunction (Concatenation (Concatenation (a,b),c),d)
true
true
false
false
Concatenation (Disjunction (a,b),c)
false
false
true
true
false
false
Disjunction (a,Disjunction (b,c))
true
true
true
false
false
Concatenation (Disjunction (a,Disjunction (b,c)),d)
true
true
true
false
false
false
false
Concatenation (Disjunction (Concatenation (a,b),c),d)
true
true
true
false
false
Concatenation (a,Disjunction (b,c))
true
true
false
false
false

